### ISSUE TITLE
The bloom filter does not get saved to the file correctly

### ISSUE THREAD

**Issue by derrandz** [prashnts/pybloomfiltermmap3#3]
Please visit this [#74](https://github.com/axiak/pybloomfiltermmap/issues/74) and continue reading

As @axiak has thought of encoding as a problem, and after having discovered your [issue #71](https://github.com/axiak/pybloomfiltermmap/issues/71) 

I thought your fork would solve the problem, but rerunning the tests mentioned in my issue in the original project, it proved it still fails.

Any help on this one?

**Comment by prashnts** [comment]
Hi,
I can reproduce this issue. I will work on a fix tonight for this.
Thanks!

**Comment by derrandz** [comment]
Thank you, I will wait for your fix. 

_Meanwhile, to save myself, I made a regular file solution with no mmap, repo available at [@HOuaghad/BloomFilter](https://github.com/HOuaghad/Bloom-Filter)_

---

During my own implementation, I ran into the same problem.

If you are using `bitarrays`, make sure to not set the size for the bitarrays when you are loading an old file, as it appends and prepends zeros all over the bitarray.

In my fork, when loading an old file, I ignore the supplied size parameter.

I hope this helps.
